src/interfaces/parser.ts

```diff
@@ -149,7 +149,11 @@ export type OptionFlag<T> = FlagBase<T, string> & OptionFlagProps & {
 }
 
 export type Definition<T> = {
-  (options: { multiple: true } & Partial<OptionFlag<T[]>>): OptionFlag<T[]>;
+  (
+    options: { multiple: true } & ({ required: true } | { default: Default<T> }) &
+      Partial<OptionFlag<T>>,
+  ): OptionFlag<T[]>;
+  (options: { multiple: true } & Partial<OptionFlag<T[]>>): OptionFlag<T[] | undefined>;
```

Comment:
@mdonnalley I think this change broke my code.

I have a command setup like:
```
class MyCommand extends Command {
  static flags: FlagInput<{ include: string[] }> = {
    include: Flags.string({
      multiple: true,
      default: [],
    }),
  }
}
```
And now it's giving me an error like: `Type 'OptionFlag<string[] | undefined>' is not assignable to type 'CompletableFlag<string[]>'`

I works if I edit the `parser.d.ts` file manually like so:
```
  (
    options: { multiple: true } & ({ required: true } | { default: Default<T[]> }) &
      Partial<OptionFlag<T[]>>,
  ): OptionFlag<T[]>;
```